### DISCUSSION TITLE
[@types/dropzone] Dropzone is not properly exported since v5.7.4

### DISCUSSION THREAD

**Discussion by emisand** [DefinitelyTyped/DefinitelyTyped#62278] [Issues with a @types package]
Currently the type definitions for dropzone are referencing version v5.7.2 of the library.
Since version v5.7.4 the developers changed the way the library is exposed so doing something like this no longer works:

```Typescript
import * as Dropzone from 'dropzone';
const dz = Dropzone;
dz.autoDiscover = false;
```

Now the autoDiscover property isn't inside the imported reference, but something like the following code should be done to access that property:

```Typescript
import * as Dropzone from 'dropzone';
const dz = Dropzone;
dz.default.autoDiscover = false;
```

See changes from v5.7.2 to v5.7.4 here: https://github.com/dropzone/dropzone/compare/v5.7.2...v5.7.4

Please update the typings to the latest version of dropzone

[a comment by an automated account is not shown]

**Comment by mlwyatt** [comment]
I don't know if you've gotten around your problem, but I have a similar issue, in regards to improper typing.

Since [dropzonev3.12](https://github.com/dropzone/dropzone/blob/v3.12.0/src/dropzone.coffee#L716), at least, the `paramName` option (on `DropzoneOptions`) could be a string or a function.

[In this library](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/f205c5c87c33cc44ca4bf7452ba4c559e57d736f/types/dropzone/index.d.ts#L78), the type is only listed as `string | undefined`.
